libs/i18n/src/lib/languages/ukrainian.ts

```diff
@@ -608,7 +608,9 @@ export const FD_LANGUAGE_UKRAINIAN: FdLanguage = {
         sortDialogSortByHeader: 'Сортувати за',
         sortDialogNotSortedLabel: '(Не відсортовано)',
         sortDialogConfirmBtnLabel: 'OK',
-        sortDialogCancelBtnLabel: 'Скасувати'
+        sortDialogCancelBtnLabel: 'Скасувати',
+        selectAllCheckboxLabel: 'Select all',
+        deselectAllCheckboxLabel: 'Deselect all'
```

Comment:
```suggestion
        selectAllCheckboxLabel: 'Вибрати все',
        deselectAllCheckboxLabel: 'Зняти вибір із усіх'
```

Although, deselect all sounds strange, it literally means "remove selection from all", @N1XUS HELP